Buffs tactical armor brute/bullet values.

Higgin opened the pull request:
:cl: 
tweak: Tactical armor plates now offer extra resistance against brute and bullet damage over regular medium plates.
/:cl:

Why? Because tactical armor doesn't have many selling points otherwise, and this makes it more of a step between regular armor and heavy armor in terms of all-around protection.

Higgin committed:
Adds slowdown to tactical plates, changes desc.

afterthought2 commented:
I mean, it's clearly superior to medium armor plates, which are its main competitors in this armor weight class. The desc, that it adds some laser resist, is quite accurate. If it's going to be buffed, it should have further downsides. Heavy armor has speed and usability debuffs, so is not a good point of comparison.

Higgin commented:
Updated with slowdown and changed the desc to reflect. I think it's good to have it be more of a middle-step between medium and heavy armor because, for what you got out of tactical armor before, it was an extra hoop and a half to get something ablatives you already had in the armory still did better - defend against lasers.

afterthought2 reviewed (approved):
This is a pretty complicated change, but I guess I'm all right with it.
